doc/user/content/guides/dbt.md

```diff
@@ -0,0 +1,276 @@
+    psql -U materialize -h localhost -p 6875 materialize
+    ```
+
+    ```sql
+    materialize=> SHOW SOURCES;
+
+           name
+    -------------------
+     market_orders_raw
+
+     materialize=> SHOW VIEWS;
+
+           name
+    -------------------
+     avg_bid
+     market_orders
+    ```
+
+That's it! From here on, Materialize makes sure that your models are **incrementally updated** as new data streams in, and that you get **fresh and correct results** with millisecond latency whenever you query your views.
+
+### Materializations
+
+dbt models are materialized as `views` by default, but can be configured to use a different materialization type through the `materialized` configuration parameter. This parameter can be set directly in the model file, using:
+
+```sql
+{{ config(materialized='materializedview') }}
+```
+
+Because Materialize is optimized for (near) real-time transformations of streaming data and the core of dbt is built around batch, the `dbt-materialize` adapter implements a few custom materialization types:
+
+{{% dbt-materializations %}}
+
+## Document and test a dbt project
+
+dbt can automatically generate [documentation](https://docs.getdbt.com/docs/building-a-dbt-project/documentation) for your project as a shareable website. This brings **data governance** to your streaming pipelines, speeding up life-saving processes like data discovery (_where_ to find _what_ data) and lineage (the path data takes from source(s) to sink(s), as well as the transformations that happen along the way).
+
+1. Optionally, create a `.yml` file with helpful [properties](https://docs.getdbt.com/reference/configs-and-properties) about your project resources (like model and column descriptions, or tests) and add it to directory where your models live:
+
+    ```yaml
+    version: 2
+
+    sources:
+      - name: public
+        description: "Public schema"
+        tables:
+          - name: market_orders_raw
+            description: "Market order data source (PubNub)"
+    models:
+      - name: market_orders
+        description: "Converts market order data to proper data types"
+
+      - name: avg_bid
+        description: "Computes the average bid price"
+        columns:
+          - name: symbol
+            description: "The stock ticker"
+            tests:
+              - not_null
+          - name: avg
+            description: "The average bid price"
+    ```
+
+1. To generate documentation for your project, run:
+
+    ```bash
+    dbt docs generate
+    ```
+
+    dbt will grab any additional project information and Materialize catalog metadata, then compile it into `.json` files (`manifest.json` and `catalog.json`, respectively) that can be used to feed the documentation website. You can find the compiled files under `/target`, in the dbt project folder.
+
+1. Launch the documentation website (by default, this command starts a web server on port 8000):
+
+    ```bash
+    dbt docs serve #--port 8001
+    ```
```

Comment:
Is it supposed to be 8000 above and 8001 here?

I would break "by default, this command starts" into a separate sentence instead of a parenthetical.

Reply:
The comment using 8001 implies that you can specify a different port than the default 8000.